### DISCUSSION TITLE
Mirror topics both Identity replication and prefix name

### DISCUSSION THREAD

**Discussion by vjvel** [strimzi/strimzi-kafka-operator#8371] [General]
Hi,

 i have requirement in the mirroring connection. Need to enable both identity name mirroring and also prefix based mirroring in the same connection..

``` 
replication.policy.class: "io.strimzi.kafka.connect.mirror.IdentityReplicationPolicy"
        replication.policy.separator: "."
    topicsPattern: "prod-web|prod-sms"
```

Is there any way we can define the policy class for the specific topics ?

**Comment by scholzj** [comment]
No. You would probably need to deploy multiple separate Mirror Makers each for only some topics and each with a different policy. Or write your own custom policy which will apply different rules for different topics.
